org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/logger/LogbackLoggerSupport.java

```diff
@@ -9,22 +9,51 @@
  */
 package org.eclipse.scout.rt.platform.logger;
 
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.eclipse.scout.rt.platform.util.Assertions;
 import org.slf4j.ILoggerFactory;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import ch.qos.logback.classic.Level;
 import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.core.spi.ConfigurationEvent.EventType;
 
 /**
+ * <p>
  * Logger support for Logback.
+ * </p>
+ * <p>
+ * With support for change tracking to automatically re-apply all changes after Logback reloads its logging
+ * configuration (e.g. configured by periodic rescan)
+ * </p>
  *
  * @since 5.2
  */
 public class LogbackLoggerSupport extends AbstractLoggerSupport {
 
   private static final Logger LOG = LoggerFactory.getLogger(LogbackLoggerSupport.class);
 
+  private volatile Map<Logger, Optional<LogLevel>> m_changeMap = new ConcurrentHashMap<>();
+
+  public LogbackLoggerSupport(ILoggerFactory factory) {
+    LoggerContext loggerContext = Assertions.assertNotNull((LoggerContext) factory, "No {} provided", LoggerContext.class);
+
+    loggerContext.addConfigurationEventListener(evt -> {
+      if (evt.getEventType() == EventType.CONFIGURATION_ENDED) {
+        clearInitialStates(); // initial states will be set again with the following setLogLevel calls (however they might have changed after reload)
+
+        Map<Logger, Optional<LogLevel>> changeMap = m_changeMap;
```

Comment:
add javadoc: "re-apply all dynamically set custom log-levels"

Reply:
Fixed in 6e0a26a3aa4980807138223d90d4956d76d2a13a